x-pack/test/functional_with_es_ssl/apps/uptime/alert_flyout.ts

```diff
@@ -7,10 +7,10 @@
 
 import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../ftr_provider_context';
+const delay = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
```

Comment:
I think we don't need to maintain our own delay function. `bluebird` exports one and [it's used](https://github.com/elastic/kibana/blob/master/x-pack/test/security_api_integration/tests/session_idle/cleanup.ts#L9) in other places.